tests/test_zing_stats.py

```diff
@@ -51,16 +51,89 @@ def test_parse_gerrit_change_message():
             'message': 'Patch Set 1: Verified+1\n\nBuild succeeded\n\n- https://zing.example.net/jenkins/job/test-check/6/ : SUCCESS in 7s'  # noqa
         },
     ]
-    msg0 = zing_stats.parse_gerrit_change_message(messages[0])
+    msg0 = zing_stats.parse_ci_job_comments(messages[0])
```

Comment:
you have swapped parse_gerrit_change_message for parse_ci_job_comments, but parse_gerrit_change_message has not been renamed, and I don't see parsse_ci_job_comments here?

Reply:
Thanks for the review, the renamed function is in https://github.com/HewlettPackard/zing-stats/pull/9/files#diff-005e9d157acc7da7735b077cb43d4429R434 ?

Reply:
For some reason the last file diff containing this change doesn't load by default.

Reply:
Ah right, yes, the main changes to zing_stats.py aren't displayed by default, Github seems to default to not displaying large diffs.

Reply:
Ah - yes I was getting confused with test_parse_gerrit_change_message .... perhaps that ought to change name also?